subgraph/config.py

```diff
@@ -35,10 +39,13 @@
         f"tokens-{Network.Fantom}": (
             "https://api.thegraph.com/subgraphs/name/badger-finance/badger-dao-tokens-fantom"
         ),
-        "fuse": "https://api.thegraph.com/subgraphs/name/badger-finance/fuse-subgraph",
+        "fuse": env_config.get_graph_api_key(
+            "badger-rewards/badger-fuse-gql-url", "SUBGRAPH_URL"),
```

Comment:
wait is this one deployed decentralised? i think i noted it as deprecated? can you dm me the actual prod endpoint url you are using for this? we might have to actually deploy it!

Reply:
Good point!

Reply:
i looked into compiling and deploying it, but it looks like it is not finished (has todo comments in it,doesnt compile in current state). i can try to fix it but need to know first that we really need it.

Reply:
currently the values are the same just now stored in the aws secret store so we can swap them out with the new URLS when ready